Demo/src/androidTest/java/com/braintreepayments/demo/test/PayPalBrowserSwitchTest.java

```diff
@@ -1,54 +0,0 @@
-package com.braintreepayments.demo.test;
-
-import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
-
-import com.braintreepayments.demo.test.utilities.TestHelper;
-
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import static com.braintreepayments.AutomatorAction.click;
-import static com.braintreepayments.AutomatorAssertion.text;
-import static com.braintreepayments.DeviceAutomator.onDevice;
-import static com.braintreepayments.UiObjectMatcher.withText;
-import static com.braintreepayments.UiObjectMatcher.withTextStartingWith;
-import static org.hamcrest.CoreMatchers.containsString;
-import static org.hamcrest.CoreMatchers.endsWith;
-
-@RunWith(AndroidJUnit4ClassRunner.class)
-public class PayPalBrowserSwitchTest extends TestHelper {
```

Comment:
I opted to remove these tests since we no longer have a mocked sandbox PayPal flow so they don't work - the PayPal flow continues to change and there are many paths (ex: app switch) that aren't included in these, so I don't think they provide much value. What do others think?

Reply:
I think removing them makes sense. We could hit the sandbox endpoints in the instrumentation tests for now. A long term solution might be to have API responses bundled in the tests for specific flows.